web/modules/custom/greet/greet.routing.yml

```diff
@@ -0,0 +1,23 @@
+greet.overview:
+  path: '/greet'
+  defaults:
+    _controller: 'Drupal\greet\Controller\GreetController::overview'
+    _title: 'Hi!'
+  requirements:
+    _permission: 'access content'
+
+greet.add:
+  path: '/greet/add'
+  defaults:
+    _controller: 'Drupal\greet\Controller\GreetController::add'
+    _title: 'Hi!!!!'
+  requirements:
+    _permission: 'greet permission 0'
```

Comment:
The name for this permission is not good. It is containing "0" and does not properly explain what the permission is about.
Please use a better name.